azurerm/internal/services/mariadb/tests/data_source_arm_mariadb_server_test.go

```diff
@@ -0,0 +1,117 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMMariaDbServer_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_mariadb_server", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMMariaDbServerDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMMariaDbServer_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMMariaDbServerExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "administrator_login", "acctestun"),
+					resource.TestCheckResourceAttr(data.ResourceName, "version", "10.2"),
+					resource.TestCheckResourceAttr(data.ResourceName, "ssl_enforcement", "Enabled"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceAzureRMMariaDbServer_basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
could we use 
```suggestion
  name     = "acctestRG-maria-%d"
```

Reply:
It won't let me commit your suggestion on this one (greyed out?), but I made the change.